spec/common/mount_tmpfs_spec.rb

```diff
@@ -0,0 +1,175 @@
+require 'spec_helper'
+
+# Test mount commands for tmpfs
+
+describe command('mkdir /mnt/tmpfs') do
```

Comment:
Actually, the tests eventually do `/mnt/tmpfs` so that's probably not really needed